zmon-common/src/main/java/org/zalando/zmon/domain/ServiceLevelStatus.java

```diff
@@ -41,47 +43,63 @@ public void setLastModifiedBy(String lastModifiedBy) {
         static final String FIFTH_LEVEL_WARNING = "Storage of metrics is currently only available for metrics classified as \"important\" or \"critical\". Metrics visualization is currently only enabled for metrics classified as \"critical\" and is temporarily limited to the last 12 hours.";
         static final String SIXTH_LEVEL_WARNING = "Metrics visualization & storage is currently only available for metrics classified as \"critical\" and is temporarily limited to the last 12 hours.";
 
-        private Integer ingestMaxCheckTier = 3;
+        private Integer ingestMaxCheckTier = 0;
```

Comment:
why are these fields nullable? It's asking for NullPointerExceptions in auto unboxing, e.g. `this.queryDistanceHoursLimit != 0`

Reply:
this is being addressed, thanks for pointing it out.